double_trans_cipher.rb

```diff
@@ -1,15 +1,59 @@
+## DoubleTranspositionCipher
 module DoubleTranspositionCipher
+  def self.row_swap(origin, dest, random_key)
+    origin.each_with_index do |_row, i|
+      dest[random_key[i]] = origin[i]
+    end
+    dest
+  end
+
+  def self.undo_swap(origin, dest, random_key)
```

Comment:
It's an empty matrix of the same size that I'm passing in, Ruby doesn't allow you to deep copy arrays. So I couldn't do a modification of the original array without loosing values that I needed